modules/analysis/rosbag-image-pipeline/README.md

```diff
@@ -0,0 +1,150 @@
+# Ros Image Extraction Pipeline
+
+## Description
```

Comment:
Can you add more notes about what this module does and its business relevance, added to the resources it deploys and how it orchestrates?

Reply:
Done